tests/console/mutt.pm

```diff
@@ -31,7 +31,7 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
-    zypper_call("in mutt", exitcode => [0, 102, 103]) if (is_tumbleweed || is_jeos);
+    zypper_call("in mutt", exitcode => [0, 102, 103]) if (is_tumbleweed || is_jeos || get_var('PUBLIC_CLOUD'));
```

Comment:
@baierjan I believe `mutt` is missing on purpose on our Public Cloud instances. Do you agree?

Reply:
Public Cloud and jeos have probably a lot in common. For example missing almost everything by default :smile: